packages/sdk/contractkit/src/kit.test.ts

```diff
@@ -150,3 +157,77 @@ describe('newKitWithApiKey()', () => {
     expect(fetchSpy).toHaveBeenCalled()
   })
 })
+
+/**
+ * These are "developer tests", meaning that they test implementation specific aspects, and might
```

Comment:
ideally we don't test implementation specific details, so my question is: what are we testing here? 

looks to me like we're essentially testing if transferring Celo works with EVERY single fee currency, correct? so I'd say it's a contractkit test and it doesn't look to me like testing implementation details

the description is not clear to me

Reply:
Actually this is an old comment from @arthurgousset let me rephrase/make sure we understand it